### ISSUE TITLE
HyperParams

### ISSUE THREAD

**Issue by alexandrosXe** [guilk/KAT#13]
Hi, very nice work in NAACL 2022. I am trying to reproduce some results. Could you please release the exact values of the hyperparams that were used in your experiments?  
-lr
-batch_size
-optimizer
-scheduler
-weight_decay
-total_steps
-warmup_steps
-scheduler_steps
-eval_freq
-max_seq_len 
-answer_maxlength
-clip_grads 

Thank you and kind regards, 
Alexandros Xenos

**Comment by guilk** [comment]
Hi, thanks for your interest in our work.

Can you check the readme file? I have listed all the hyperparameters to reproduce the results.

**Comment by alexandrosXe** [comment]
Thank you for your quick response! 
Oh, I thought that these are not the real values since in the paper it says "We use a learning rate of 3e − 5
to warm up for 2K iterations and train for 10K iterations." and in the readme file, it says warmup_steps = 1000 and total_steps = 8000. Also, I suppose for the hyperparams that are not listed there, the default values were used, right?

**Comment by guilk** [comment]
Sorry for the confusion. As far as I can remember, the model is not that sensitive to the learning rate, warmup steps and total steps if the hyperparameters are in a reasonable range. I tested the learning rate from 1e-5 to 3e-5 and got similar results.
